superset-frontend/spec/javascripts/explore/utils_spec.jsx

```diff
@@ -202,20 +201,6 @@ describe('exploreUtils', () => {
     });
   });
 
-  describe('getDataTablePageSize', () => {
-    it('divides samples data into pages dynamically', () => {
-      let pageSize;
-      pageSize = getDataTablePageSize(500);
-      expect(pageSize).toEqual(20);
-      pageSize = getDataTablePageSize(0);
-      expect(pageSize).toEqual(50);
```

Comment:
Can we keep this test since we now got a constant for items per page?

Reply:
Do we actually need this test? We use a fixed number of items per page now and pagination itself is tested in `TableView_spec.tsx`

Reply:
Ah got it